candi/openapi/doc-ru-node_group.yaml

```
@@ -30,9 +30,9 @@ spec:
                     type:
                     containerd:
                       description: |
-                        Параметры работы Containerd.
+                        Параметры работы containerd.
 
-                        При настройке этих параметров `cri.type` должен быть — `Containerd`.
+                        При настройке этих параметров `cri.type` должен быть `containerd`.
```

```suggestion
                        При настройке этих параметров `cri.type` должен быть `Containerd`.
```
The value of the cri.type can be ane of `Docker`, `Containerd`